lib/driver-helper.js

```diff
@@ -416,3 +416,12 @@ export async function scrollIntoView( driver, selector ) {
 		selectorElement
 	);
 }
+
+export async function scrollToBottom( driver, selector ) {
```

Comment:
`scrollIntoView()` didn't work always because Viewers list is infinite and users didn't load all at once. That's why I needed a way to scroll list(or, generally, any other element) step by step.